src/main/java/task/Task.java

```diff
@@ -0,0 +1,43 @@
+package task;
+
+public class Task {
+    private int hasCompleted;
```

Comment:
For such a variable name, and for its purpose and description in your code, would a boolean be a more appropriate type for this variable?

Reply:
I stored the variable type of int because the txt file uses integer for storage, I'll rename the variable to hasCompletedInt, thanks!